gangplank/internal/ocp/return.go

```diff
@@ -98,10 +98,11 @@ func (r *Return) Run(ctx context.Context, ws *workSpec) error {
 	}
 
 	// Now any kola logs.
-	tmpFiles, _ := ioutil.ReadDir(filepath.Join(cosaSrvDir, "tmp"))
+	tmpDir := filepath.Join(cosaSrvDir, "tmp")
+	tmpFiles, _ := ioutil.ReadDir(tmpDir)
 	for _, f := range tmpFiles {
 		upKey := filepath.Join(keyPath, "logs", f.Name())
-		srcPath := filepath.Join(path, f.Name())
+		srcPath := filepath.Join(tmpDir, f.Name())
 		if strings.Contains(f.Name(), "kola") && strings.HasSuffix(f.Name(), "tar.xz") {
```

Comment:
Aside: this looks like it's only passing back compressed tarballs; does this mean gangplank knows to automatically compress into tarballs after kola steps? (Related: https://github.com/coreos/fedora-coreos-pipeline/commit/c3a57a9ccf8ff72f2b253b4145277ccc593f9a45.)

Reply:
yeah I think gangplank does - but i'm kind of doing it manually myself right now as some of gangplanks automation gets in the way of what I need. PR for that incoming soon.